pages/death.vue

```diff
@@ -251,7 +209,6 @@
            // push
            if (witch) {
              if (id === "http://www.wikidata.org/entity/Q43394934") {
```

Comment:
Please remove, as this doesn't do anything

Reply:
Resolved.